src/Pos/Communication/Message.hs

```diff
@@ -94,9 +94,17 @@ instance (MessagePart tag) =>
         tagM _ = Proxy
     formatMessage _ = "Request"
 
+instance (MessagePart tag) =>
```

Comment:
Should I do it or should I create an issue or what? If the answer is “@neongreen should do it” – can you point me at the file that needs to be changed?